src/Tribe/Promoter/Triggers/Builders/Attendee_Trigger.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+
+namespace Tribe\Tickets\Promoter\Triggers\Builders;
+
```

Comment:
:no_entry_sign: There must be one blank line after the namespace declaration

<sub>hash: a438f5c373d13ad3503dd12f53b1373f</sub>